src/features/advanced-content-insights/ContentInsightsSection.js

```diff
@@ -0,0 +1,145 @@
+// @flow
```

Comment:
since this is a brand new file, could we write this in TypeScript instead of JS + flow?

Reply:
I changed the file's name to AdvancedContentInsightsToggle. We received a last update to drop the options, so it only has the toggle for now.